app/cdap/components/ParsingDrawer/Components/ParsingHeaderActionTemplate/index.tsx

```diff
@@ -16,19 +16,16 @@
 
 import { Box } from '@material-ui/core';
 import { useStyles } from 'components/ParsingDrawer/styles';
-import React, { ChangeEvent } from 'react';
+import React from 'react';
 import { parseImportedSchemas } from 'components/AbstractWidget/SchemaEditor/SchemaHelpers';
 import T from 'i18n-react';
 
-export default function(props) {
+export default function(props){
```

Comment:
expand props and use interface if we are using here

Reply:
Done